packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -551,6 +559,9 @@ export class DashboardModel {
                 dashboard.dashboard_version_id,
             );
 
+        const tableCalculationFilters = view?.filters?.tableCalculations;
+        view.filters.tableCalculations = tableCalculationFilters || [];
```

Comment:
Is this necessary ? It feels a bit redundant. I don't see it happening for other filters like `dimensions` or `metrics`

Reply:
Hi Rephus, reason doing so is because dashbaord view is stored as json in the database, so i'm not sure what's the recommended way to migrate the json object to include default table calculation filters list, so instead i set it to empty list here if its not exist in the database.